stark-brainfuck/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "stark-brainfuck"
+version = "0.2.0"
+authors = ["Triton Software AG"]
+edition = "2021"
+
+license = "GPL-2.0"
+description = "A Turing-complete zk-STARK engine consisting of a virtual machine, prover and verifier."
+homepage = "https://github.com/Neptune-Crypto/twenty-first"
```

Comment:
Would it make sense to link to the tutorial here?

Reply:
Yes!